src/main/resources/graphql/resource.graphqls

```diff
@@ -0,0 +1,24 @@
+"""
+Resources are all types of content present in a course. Each resource can be available or unavailable in a course.
+"""
+type Resource {
```

Comment:
I'm not completely happy with just naming this "Resource", as this type cannot be used to actually fetch or do anything with the resource. Maybe something like "ResourceInformation"...?

Reply:
I agree with this naming being suboptimal/misleading, but could not think of a better name to describe the Entity

Reply:
![image](https://github.com/IT-REX-Platform/course_service/assets/34026908/a617b35f-c386-4482-b582-59bb24f885f1)